themes/dco/templates/individual--foaf-person.ftl

```diff
@@ -49,6 +49,9 @@
                 
         <!-- Websites -->
         <#include "individual-webpage.ftl">
+        
+        <!-- Nationality -->
+        <!-- Nationality: ${homeCountry!}  -->
```

Comment:
Then we should probably comment out both with <#-- stuff commented out --> so the FTL template engine does not attempt to resolve ${homeCountry!}

XML/HTML comment: <!-- not shown in browser -->
FTL comment: <#-- not processed by FTL engine -->